  INFO root [0x00001524] [2018-05-06 13:45:27,640] - 
  INFO root [0x00001524] [2018-05-06 13:45:27,691] - -----------------------------------------------------------------------------------------


  INFO root [0x00001524] [2018-05-06 13:45:28,569] - Resmgr::initialize: KBE_ROOT=E:/kbengine/
  INFO root [0x00001524] [2018-05-06 13:45:28,569] - Resmgr::initialize: KBE_RES_PATH=E:/kbengine/kbe/res/;E:/kbengine/assets/;E:/kbengine/assets/scripts/;E:/kbengine/assets/res/
  INFO root [0x00001524] [2018-05-06 13:45:28,569] - Resmgr::initialize: KBE_BIN_PATH=E:/kbengine/kbe/bin/server/
  INFO root [0x00001524] [2018-05-06 13:45:29,462] - NetworkInterface::initialize(INTERNAL): bound to all interfaces with default route interface on 192.168.0.106 ( 192.168.0.106:50286 )
  INFO root [0x00001524] [2018-05-06 13:45:29,509] - NetworkInterface::initialize(INTERNAL): address 192.168.0.106:50286, SOMAXCONN=5.
  INFO root [0x00001524] [2018-05-06 13:45:29,509] - server-configs:
  INFO root [0x00001524] [2018-05-06 13:45:29,510] - 	gameUpdateHertz : 10
  INFO root [0x00001524] [2018-05-06 13:45:29,543] - 	defaultViewRadius : 80
  INFO root [0x00001524] [2018-05-06 13:45:29,543] - 	defaultViewHysteresisArea : 5
  INFO root [0x00001524] [2018-05-06 13:45:29,544] - 	entryScriptFile : kbemain
  INFO root [0x00001524] [2018-05-06 13:45:29,544] - 	internalAddr : 192.168.0.106:50286
  INFO root [0x00001524] [2018-05-06 13:45:29,544] - 	componentID : 8001
  INFO root [0x00001524] [2018-05-06 13:45:29,906] - Components::findLogger: found logger, addr:192.168.0.106:50274
  INFO root [0x00001524] [2018-05-06 13:45:29,907] - Components::addComponent: reset orderLog, uid=4706!
  INFO root [0x00001524] [2018-05-06 13:45:29,938] - Components::addComponent[logger], uid=4706, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [0x00001524] [2018-05-06 13:45:30,157] - ---- cellapp Version: 2.0.0. ScriptVersion: 0.1.0. Protocol: 85A70C5E64F896E0F1A60BF405B4EB07. Config: Debug X86. Built: 15:53:53 Apr 29 2018. AppID: 8001. UID: 4706. PID: 4932 ----
  INFO root [0x00001524] [2018-05-06 13:45:30,158] - Server : Intel(R) Xeon(R) CPU E3-1230 v3 @ 3.30GHz with 4194303kB RAM
  INFO root [0x00001524] [2018-05-06 13:45:30,268] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [0x00001524] [2018-05-06 13:45:30,302] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
 DEBUG root [0x00001524] [2018-05-06 13:45:36,495] - Script::install_py_dlls(): PyInit_pyexpat
 DEBUG root [0x00001524] [2018-05-06 13:45:36,676] - Script::install_py_dlls(): PyInit__socket
 DEBUG root [0x00001524] [2018-05-06 13:45:36,865] - Script::install_py_dlls(): PyInit__ssl
 DEBUG root [0x00001524] [2018-05-06 13:45:38,093] - Script::install_py_dlls(): PyInit__hashlib
 DEBUG root [0x00001524] [2018-05-06 13:45:38,283] - Script::install_py_dlls(): PyInit_select
 DEBUG root [0x00001524] [2018-05-06 13:45:38,355] - Script::install_py_dlls(): PyInit__ctypes
 DEBUG root [0x00001524] [2018-05-06 13:45:38,421] - Script::install_py_dlls(): PyInit__elementtree
 DEBUG root [0x00001524] [2018-05-06 13:45:38,846] - Script::install_py_dlls(): PyInit_unicodedata
  INFO root [0x00001524] [2018-05-06 13:45:39,852] - Script::install(): is successfully, Python=(3.4.2 (default, Apr 29 2018, 15:53:05) [MSC v.1913 32 bit (Intel)])!
  INFO root [0x00001524] [2018-05-06 13:45:39,853] - Script::install(): KBExtra is successfully!
  INFO root [0x00001524] [2018-05-06 13:45:40,683] - TelnetServer server is running on port 50000
  INFO root [0x00001524] [2018-05-06 13:45:40,987] - ---- cellapp is running ----
 DEBUG root [0x00001524] [2018-05-06 13:45:40,987] - Components::process(): Request for the process of identity...
  WARN root [0x00001524] [2018-05-06 13:45:47,080] - Forwarding logs to logger[192.168.0.106:50274/0/0/0]...
 ERROR root [0x00001524] [2018-05-06 13:46:05,862] - Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  INFO root [0x00001524] [2018-05-06 13:46:05,872] - The following logs sent to logger failed:
  WARN root [0x00001524] [2018-05-06 13:46:05,873] - ==> [2018-05-06 13:46:05 860] TCPPacketReceiver::processPendingEvents(192.168.0.106:50274): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x00001524] [2018-05-06 13:46:05,873] - ==> [2018-05-06 13:46:05 863] Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  WARN root [0x00001524] [2018-05-06 13:46:05,916] - TCPPacketReceiver::processPendingEvents(192.168.0.106:50277): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x00001524] [2018-05-06 13:46:05,918] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit! errorno=10054
  INFO root [0x00001524] [2018-05-06 13:46:05,939] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [0x00001524] [2018-05-06 13:46:05,940] - Shutdowner::onShutdown: shutting down
  INFO root [0x00001524] [2018-05-06 13:46:05,940] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [0x00001524] [2018-05-06 13:46:06,082] - EntityRef::destroyObjPool(): size 0.
 DEBUG root [0x00001524] [2018-05-06 13:46:06,082] - Witness::destroyObjPool(): size 0.
  INFO root [0x00001484] [2018-05-06 13:46:07,114] - 
  INFO root [0x00001484] [2018-05-06 13:46:07,116] - -----------------------------------------------------------------------------------------


  INFO root [0x00001484] [2018-05-06 13:46:07,117] - Resmgr::initialize: KBE_ROOT=E:/kbengine/
  INFO root [0x00001484] [2018-05-06 13:46:07,117] - Resmgr::initialize: KBE_RES_PATH=E:/kbengine/kbe/res/;E:/kbengine/assets/;E:/kbengine/assets/scripts/;E:/kbengine/assets/res/
  INFO root [0x00001484] [2018-05-06 13:46:07,117] - Resmgr::initialize: KBE_BIN_PATH=E:/kbengine/kbe/bin/server/
  INFO root [0x00001484] [2018-05-06 13:46:07,127] - NetworkInterface::initialize(INTERNAL): bound to all interfaces with default route interface on 192.168.0.106 ( 192.168.0.106:50311 )
  INFO root [0x00001484] [2018-05-06 13:46:07,128] - NetworkInterface::initialize(INTERNAL): address 192.168.0.106:50311, SOMAXCONN=5.
  INFO root [0x00001484] [2018-05-06 13:46:07,128] - server-configs:
  INFO root [0x00001484] [2018-05-06 13:46:07,128] - 	gameUpdateHertz : 10
  INFO root [0x00001484] [2018-05-06 13:46:07,128] - 	defaultViewRadius : 80
  INFO root [0x00001484] [2018-05-06 13:46:07,128] - 	defaultViewHysteresisArea : 5
  INFO root [0x00001484] [2018-05-06 13:46:07,128] - 	entryScriptFile : kbemain
  INFO root [0x00001484] [2018-05-06 13:46:07,128] - 	internalAddr : 192.168.0.106:50311
  INFO root [0x00001484] [2018-05-06 13:46:07,128] - 	componentID : 8001
  WARN root [0x00001484] [2018-05-06 13:46:07,439] - BundleBroadcast::BundleBroadcast: Cannot bind to port 20275, errorno=10048
  INFO root [0x00001484] [2018-05-06 13:46:07,445] - ---- cellapp Version: 2.0.0. ScriptVersion: 0.1.0. Protocol: 85A70C5E64F896E0F1A60BF405B4EB07. Config: Debug X86. Built: 15:53:53 Apr 29 2018. AppID: 8001. UID: 4915. PID: 9052 ----
  INFO root [0x00001484] [2018-05-06 13:46:07,445] - Server : Intel(R) Xeon(R) CPU E3-1230 v3 @ 3.30GHz with 4194303kB RAM
  INFO root [0x00001484] [2018-05-06 13:46:07,446] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [0x00001484] [2018-05-06 13:46:07,446] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
 DEBUG root [0x00001484] [2018-05-06 13:46:07,577] - Script::install_py_dlls(): PyInit_pyexpat
 DEBUG root [0x00001484] [2018-05-06 13:46:07,577] - Script::install_py_dlls(): PyInit__socket
 DEBUG root [0x00001484] [2018-05-06 13:46:07,578] - Script::install_py_dlls(): PyInit__ssl
 DEBUG root [0x00001484] [2018-05-06 13:46:07,581] - Script::install_py_dlls(): PyInit__hashlib
 DEBUG root [0x00001484] [2018-05-06 13:46:07,581] - Script::install_py_dlls(): PyInit_select
 DEBUG root [0x00001484] [2018-05-06 13:46:07,581] - Script::install_py_dlls(): PyInit__ctypes
 DEBUG root [0x00001484] [2018-05-06 13:46:07,582] - Script::install_py_dlls(): PyInit__elementtree
 DEBUG root [0x00001484] [2018-05-06 13:46:07,611] - Script::install_py_dlls(): PyInit_unicodedata
  INFO root [0x00001484] [2018-05-06 13:46:07,641] - Script::install(): is successfully, Python=(3.4.2 (default, Apr 29 2018, 15:53:05) [MSC v.1913 32 bit (Intel)])!
  INFO root [0x00001484] [2018-05-06 13:46:07,641] - Script::install(): KBExtra is successfully!
  INFO root [0x00001484] [2018-05-06 13:46:07,652] - TelnetServer server is running on port 50000
  INFO root [0x00001484] [2018-05-06 13:46:07,979] - ---- cellapp is running ----
 DEBUG root [0x00001484] [2018-05-06 13:46:07,979] - Components::process(): Request for the process of identity...
  WARN root [0x00001484] [2018-05-06 13:46:14,048] - EntityApp::checkTickPeriod: tick took 6068% (6.07 seconds)!
  INFO root [0x00001484] [2018-05-06 13:46:15,081] - Components::findComponents: find logger(1)...
  INFO root [0x00001484] [2018-05-06 13:46:15,128] - Components::findComponents: found logger, addr:192.168.0.106:50316
  INFO root [0x00001484] [2018-05-06 13:46:15,128] - Components::addComponent: reset orderLog, uid=4915!
  INFO root [0x00001484] [2018-05-06 13:46:15,129] - Components::addComponent[logger], uid=4915, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [0x00001484] [2018-05-06 13:46:15,131] - Components::findComponents: find dbmgr(2)...
  INFO root [0x00001484] [2018-05-06 13:46:15,233] - Components::findComponents: found dbmgr, addr:192.168.0.106:50310
  INFO root [0x00001484] [2018-05-06 13:46:15,233] - Components::addComponent: reset orderLog, uid=4915!
  INFO root [0x00001484] [2018-05-06 13:46:15,234] - Components::addComponent[dbmgr], uid=4915, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [0x00001484] [2018-05-06 13:46:15,235] - Components::findComponents: found dbmgr, addr:192.168.0.106:50310
  WARN root [0x00001484] [2018-05-06 13:46:16,235] - EntityApp::checkTickPeriod: tick took 1254% (1.25 seconds)!
  WARN root [0x00001484] [2018-05-06 13:46:16,235] - Forwarding logs to logger[192.168.0.106:50316/0/0/0]...
 ERROR root [0x00001484] [2018-05-06 13:51:34,950] - Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  INFO root [0x00001484] [2018-05-06 13:51:34,958] - The following logs sent to logger failed:
  WARN root [0x00001484] [2018-05-06 13:51:34,958] - ==> [2018-05-06 13:51:34 945] TCPPacketReceiver::processPendingEvents(192.168.0.106:50316): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x00001484] [2018-05-06 13:51:34,958] - ==> [2018-05-06 13:51:34 951] Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  WARN root [0x00001484] [2018-05-06 13:51:35,005] - TCPPacketReceiver::processPendingEvents(192.168.0.106:50310): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x00001484] [2018-05-06 13:51:35,007] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit! errorno=10054
  INFO root [0x00001484] [2018-05-06 13:51:35,013] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [0x00001484] [2018-05-06 13:51:35,015] - Shutdowner::onShutdown: shutting down
  INFO root [0x00001484] [2018-05-06 13:51:35,015] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [0x00001484] [2018-05-06 13:51:35,016] - EntityRef::destroyObjPool(): size 0.
 DEBUG root [0x00001484] [2018-05-06 13:51:35,016] - Witness::destroyObjPool(): size 0.
 DEBUG root [0x00001484] [2018-05-06 13:51:35,044] - Script::uninstall_py_dlls(): PyInit_pyexpat
 DEBUG root [0x00001484] [2018-05-06 13:51:35,045] - Script::uninstall_py_dlls(): PyInit__socket
 DEBUG root [0x00001484] [2018-05-06 13:51:35,045] - Script::uninstall_py_dlls(): PyInit__ssl
 DEBUG root [0x00001484] [2018-05-06 13:51:35,045] - Script::uninstall_py_dlls(): PyInit__hashlib
 DEBUG root [0x00001484] [2018-05-06 13:51:35,045] - Script::uninstall_py_dlls(): PyInit_select
 DEBUG root [0x00001484] [2018-05-06 13:51:35,045] - Script::uninstall_py_dlls(): PyInit__ctypes
 DEBUG root [0x00001484] [2018-05-06 13:51:35,045] - Script::uninstall_py_dlls(): PyInit__elementtree
 DEBUG root [0x00001484] [2018-05-06 13:51:35,045] - Script::uninstall_py_dlls(): PyInit_unicodedata
  INFO root [0x00001484] [2018-05-06 13:51:35,060] - Script::uninstall(): is successfully!
 DEBUG root [0x00001484] [2018-05-06 13:51:35,061] - ThreadPool::destroy(): starting size 2.
  INFO root [0x00001470] [2018-05-06 13:51:36,127] - 
  INFO root [0x00001470] [2018-05-06 13:51:36,127] - -----------------------------------------------------------------------------------------


  INFO root [0x00001470] [2018-05-06 13:51:36,128] - Resmgr::initialize: KBE_ROOT=E:/kbengine/
  INFO root [0x00001470] [2018-05-06 13:51:36,128] - Resmgr::initialize: KBE_RES_PATH=E:/kbengine/kbe/res/;E:/kbengine/assets/;E:/kbengine/assets/scripts/;E:/kbengine/assets/res/
  INFO root [0x00001470] [2018-05-06 13:51:36,128] - Resmgr::initialize: KBE_BIN_PATH=E:/kbengine/kbe/bin/server/
  INFO root [0x00001470] [2018-05-06 13:51:36,130] - NetworkInterface::initialize(INTERNAL): bound to all interfaces with default route interface on 192.168.0.106 ( 192.168.0.106:50405 )
  INFO root [0x00001470] [2018-05-06 13:51:36,130] - NetworkInterface::initialize(INTERNAL): address 192.168.0.106:50405, SOMAXCONN=5.
  INFO root [0x00001470] [2018-05-06 13:51:36,130] - server-configs:
  INFO root [0x00001470] [2018-05-06 13:51:36,130] - 	gameUpdateHertz : 10
  INFO root [0x00001470] [2018-05-06 13:51:36,130] - 	defaultViewRadius : 80
  INFO root [0x00001470] [2018-05-06 13:51:36,130] - 	defaultViewHysteresisArea : 5
  INFO root [0x00001470] [2018-05-06 13:51:36,130] - 	entryScriptFile : kbemain
  INFO root [0x00001470] [2018-05-06 13:51:36,130] - 	internalAddr : 192.168.0.106:50405
  INFO root [0x00001470] [2018-05-06 13:51:36,131] - 	componentID : 8001
  INFO root [0x00001470] [2018-05-06 13:51:37,643] - ---- cellapp Version: 2.0.0. ScriptVersion: 0.1.0. Protocol: 85A70C5E64F896E0F1A60BF405B4EB07. Config: Debug X86. Built: 15:53:53 Apr 29 2018. AppID: 8001. UID: 5989. PID: 1652 ----
  INFO root [0x00001470] [2018-05-06 13:51:37,644] - Server : Intel(R) Xeon(R) CPU E3-1230 v3 @ 3.30GHz with 4194303kB RAM
  INFO root [0x00001470] [2018-05-06 13:51:37,645] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [0x00001470] [2018-05-06 13:51:37,646] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
 DEBUG root [0x00001470] [2018-05-06 13:51:37,791] - Script::install_py_dlls(): PyInit_pyexpat
 DEBUG root [0x00001470] [2018-05-06 13:51:37,792] - Script::install_py_dlls(): PyInit__socket
 DEBUG root [0x00001470] [2018-05-06 13:51:37,792] - Script::install_py_dlls(): PyInit__ssl
 DEBUG root [0x00001470] [2018-05-06 13:51:37,795] - Script::install_py_dlls(): PyInit__hashlib
 DEBUG root [0x00001470] [2018-05-06 13:51:37,796] - Script::install_py_dlls(): PyInit_select
 DEBUG root [0x00001470] [2018-05-06 13:51:37,796] - Script::install_py_dlls(): PyInit__ctypes
 DEBUG root [0x00001470] [2018-05-06 13:51:37,796] - Script::install_py_dlls(): PyInit__elementtree
 DEBUG root [0x00001470] [2018-05-06 13:51:37,824] - Script::install_py_dlls(): PyInit_unicodedata
  INFO root [0x00001470] [2018-05-06 13:51:37,852] - Script::install(): is successfully, Python=(3.4.2 (default, Apr 29 2018, 15:53:05) [MSC v.1913 32 bit (Intel)])!
  INFO root [0x00001470] [2018-05-06 13:51:37,852] - Script::install(): KBExtra is successfully!
 DEBUG root [0x00001470] [2018-05-06 13:51:37,887] - loaded entity-script:Account(1).
  INFO root [0x00001470] [2018-05-06 13:51:37,888] - TelnetServer server is running on port 50000
  INFO root [0x00001470] [2018-05-06 13:51:38,245] - ---- cellapp is running ----
 DEBUG root [0x00001470] [2018-05-06 13:51:38,245] - Components::process(): Request for the process of identity...
  WARN root [0x00001470] [2018-05-06 13:51:44,316] - EntityApp::checkTickPeriod: tick took 6069% (6.07 seconds)!
  INFO root [0x00001470] [2018-05-06 13:51:45,347] - Components::findComponents: find logger(1)...
  INFO root [0x00001470] [2018-05-06 13:51:45,384] - Components::findComponents: found logger, addr:192.168.0.106:50400
  INFO root [0x00001470] [2018-05-06 13:51:45,384] - Components::addComponent: reset orderLog, uid=5989!
  INFO root [0x00001470] [2018-05-06 13:51:45,385] - Components::addComponent[logger], uid=5989, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [0x00001470] [2018-05-06 13:51:45,387] - Components::findComponents: find dbmgr(2)...
  WARN root [0x00001470] [2018-05-06 13:51:46,488] - EntityApp::checkTickPeriod: tick took 1241% (1.24 seconds)!
  WARN root [0x00001470] [2018-05-06 13:51:46,490] - Forwarding logs to logger[192.168.0.106:50400/0/0/0]...
  INFO root [0x00001d28] [2018-05-06 13:51:51,530] - 
  INFO root [0x00001d28] [2018-05-06 13:51:51,531] - -----------------------------------------------------------------------------------------


  INFO root [0x00001d28] [2018-05-06 13:51:51,531] - Resmgr::initialize: KBE_ROOT=E:/kbengine/
  INFO root [0x00001d28] [2018-05-06 13:51:51,532] - Resmgr::initialize: KBE_RES_PATH=E:/kbengine/kbe/res/;E:/kbengine/assets/;E:/kbengine/assets/scripts/;E:/kbengine/assets/res/
  INFO root [0x00001d28] [2018-05-06 13:51:51,532] - Resmgr::initialize: KBE_BIN_PATH=E:/kbengine/kbe/bin/server/
  INFO root [0x00001d28] [2018-05-06 13:51:51,540] - NetworkInterface::initialize(INTERNAL): bound to all interfaces with default route interface on 192.168.0.106 ( 192.168.0.106:50429 )
  INFO root [0x00001d28] [2018-05-06 13:51:51,540] - NetworkInterface::initialize(INTERNAL): address 192.168.0.106:50429, SOMAXCONN=5.
  INFO root [0x00001d28] [2018-05-06 13:51:51,541] - server-configs:
  INFO root [0x00001d28] [2018-05-06 13:51:51,541] - 	gameUpdateHertz : 10
  INFO root [0x00001d28] [2018-05-06 13:51:51,541] - 	defaultViewRadius : 80
  INFO root [0x00001d28] [2018-05-06 13:51:51,541] - 	defaultViewHysteresisArea : 5
  INFO root [0x00001d28] [2018-05-06 13:51:51,541] - 	entryScriptFile : kbemain
  INFO root [0x00001d28] [2018-05-06 13:51:51,541] - 	internalAddr : 192.168.0.106:50429
  INFO root [0x00001d28] [2018-05-06 13:51:51,541] - 	componentID : 8001
  INFO root [0x00001d28] [2018-05-06 13:51:53,050] - ---- cellapp Version: 2.0.0. ScriptVersion: 0.1.0. Protocol: 85A70C5E64F896E0F1A60BF405B4EB07. Config: Debug X86. Built: 15:53:53 Apr 29 2018. AppID: 8001. UID: 6042. PID: 11036 ----
  INFO root [0x00001d28] [2018-05-06 13:51:53,051] - Server : Intel(R) Xeon(R) CPU E3-1230 v3 @ 3.30GHz with 4194303kB RAM
  INFO root [0x00001d28] [2018-05-06 13:51:53,051] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [0x00001d28] [2018-05-06 13:51:53,052] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
 DEBUG root [0x00001d28] [2018-05-06 13:51:53,216] - Script::install_py_dlls(): PyInit_pyexpat
 DEBUG root [0x00001d28] [2018-05-06 13:51:53,216] - Script::install_py_dlls(): PyInit__socket
 DEBUG root [0x00001d28] [2018-05-06 13:51:53,217] - Script::install_py_dlls(): PyInit__ssl
 DEBUG root [0x00001d28] [2018-05-06 13:51:53,219] - Script::install_py_dlls(): PyInit__hashlib
 DEBUG root [0x00001d28] [2018-05-06 13:51:53,220] - Script::install_py_dlls(): PyInit_select
 DEBUG root [0x00001d28] [2018-05-06 13:51:53,220] - Script::install_py_dlls(): PyInit__ctypes
 DEBUG root [0x00001d28] [2018-05-06 13:51:53,220] - Script::install_py_dlls(): PyInit__elementtree
 DEBUG root [0x00001d28] [2018-05-06 13:51:53,248] - Script::install_py_dlls(): PyInit_unicodedata
  INFO root [0x00001d28] [2018-05-06 13:51:53,277] - Script::install(): is successfully, Python=(3.4.2 (default, Apr 29 2018, 15:53:05) [MSC v.1913 32 bit (Intel)])!
  INFO root [0x00001d28] [2018-05-06 13:51:53,277] - Script::install(): KBExtra is successfully!
 DEBUG root [0x00001d28] [2018-05-06 13:51:53,290] - loaded entity-script:Account(1).
  INFO root [0x00001d28] [2018-05-06 13:51:53,291] - TelnetServer server is running on port 50000
  INFO root [0x00001d28] [2018-05-06 13:51:53,671] - ---- cellapp is running ----
 DEBUG root [0x00001d28] [2018-05-06 13:51:53,671] - Components::process(): Request for the process of identity...
  WARN root [0x00001d28] [2018-05-06 13:51:59,757] - EntityApp::checkTickPeriod: tick took 6085% (6.09 seconds)!
  INFO root [0x00001d28] [2018-05-06 13:52:00,833] - Components::findComponents: find logger(1)...
  INFO root [0x00001d28] [2018-05-06 13:52:00,932] - Components::findComponents: found logger, addr:192.168.0.106:50428
  INFO root [0x00001d28] [2018-05-06 13:52:00,933] - Components::addComponent: reset orderLog, uid=6042!
  INFO root [0x00001d28] [2018-05-06 13:52:00,933] - Components::addComponent[logger], uid=6042, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [0x00001d28] [2018-05-06 13:52:00,935] - Components::findComponents: find dbmgr(2)...
  WARN root [0x00001d28] [2018-05-06 13:52:02,044] - Forwarding logs to logger[192.168.0.106:50428/0/0/0]...
  INFO root [0x00002390] [2018-05-06 13:52:05,642] - 
  INFO root [0x00002390] [2018-05-06 13:52:05,642] - -----------------------------------------------------------------------------------------


  INFO root [0x00002390] [2018-05-06 13:52:05,643] - Resmgr::initialize: KBE_ROOT=E:/kbengine/
  INFO root [0x00002390] [2018-05-06 13:52:05,643] - Resmgr::initialize: KBE_RES_PATH=E:/kbengine/kbe/res/;E:/kbengine/assets/;E:/kbengine/assets/scripts/;E:/kbengine/assets/res/
  INFO root [0x00002390] [2018-05-06 13:52:05,643] - Resmgr::initialize: KBE_BIN_PATH=E:/kbengine/kbe/bin/server/
  INFO root [0x00002390] [2018-05-06 13:52:05,648] - NetworkInterface::initialize(INTERNAL): bound to all interfaces with default route interface on 192.168.0.106 ( 192.168.0.106:50451 )
  INFO root [0x00002390] [2018-05-06 13:52:05,648] - NetworkInterface::initialize(INTERNAL): address 192.168.0.106:50451, SOMAXCONN=5.
  INFO root [0x00002390] [2018-05-06 13:52:05,648] - server-configs:
  INFO root [0x00002390] [2018-05-06 13:52:05,648] - 	gameUpdateHertz : 10
  INFO root [0x00002390] [2018-05-06 13:52:05,649] - 	defaultViewRadius : 80
  INFO root [0x00002390] [2018-05-06 13:52:05,649] - 	defaultViewHysteresisArea : 5
  INFO root [0x00002390] [2018-05-06 13:52:05,649] - 	entryScriptFile : kbemain
  INFO root [0x00002390] [2018-05-06 13:52:05,649] - 	internalAddr : 192.168.0.106:50451
  INFO root [0x00002390] [2018-05-06 13:52:05,649] - 	componentID : 8001
  WARN root [0x00002390] [2018-05-06 13:52:05,960] - BundleBroadcast::BundleBroadcast: Cannot bind to port 20405, errorno=10048
  INFO root [0x00002390] [2018-05-06 13:52:05,964] - ---- cellapp Version: 2.0.0. ScriptVersion: 0.1.0. Protocol: 85A70C5E64F896E0F1A60BF405B4EB07. Config: Debug X86. Built: 15:53:53 Apr 29 2018. AppID: 8001. UID: 6087. PID: 5292 ----
  INFO root [0x00002390] [2018-05-06 13:52:05,964] - Server : Intel(R) Xeon(R) CPU E3-1230 v3 @ 3.30GHz with 4194303kB RAM
  INFO root [0x00002390] [2018-05-06 13:52:05,965] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [0x00002390] [2018-05-06 13:52:05,965] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
 DEBUG root [0x00002390] [2018-05-06 13:52:06,091] - Script::install_py_dlls(): PyInit_pyexpat
 DEBUG root [0x00002390] [2018-05-06 13:52:06,091] - Script::install_py_dlls(): PyInit__socket
 DEBUG root [0x00002390] [2018-05-06 13:52:06,092] - Script::install_py_dlls(): PyInit__ssl
 DEBUG root [0x00002390] [2018-05-06 13:52:06,095] - Script::install_py_dlls(): PyInit__hashlib
 DEBUG root [0x00002390] [2018-05-06 13:52:06,095] - Script::install_py_dlls(): PyInit_select
 DEBUG root [0x00002390] [2018-05-06 13:52:06,095] - Script::install_py_dlls(): PyInit__ctypes
 DEBUG root [0x00002390] [2018-05-06 13:52:06,096] - Script::install_py_dlls(): PyInit__elementtree
 DEBUG root [0x00002390] [2018-05-06 13:52:06,122] - Script::install_py_dlls(): PyInit_unicodedata
  INFO root [0x00002390] [2018-05-06 13:52:06,149] - Script::install(): is successfully, Python=(3.4.2 (default, Apr 29 2018, 15:53:05) [MSC v.1913 32 bit (Intel)])!
  INFO root [0x00002390] [2018-05-06 13:52:06,149] - Script::install(): KBExtra is successfully!
 DEBUG root [0x00002390] [2018-05-06 13:52:06,157] - loaded entity-script:Account(1).
  INFO root [0x00002390] [2018-05-06 13:52:06,158] - TelnetServer server is running on port 50000
  INFO root [0x00002390] [2018-05-06 13:52:06,441] - ---- cellapp is running ----
 DEBUG root [0x00002390] [2018-05-06 13:52:06,441] - Components::process(): Request for the process of identity...
  WARN root [0x00002390] [2018-05-06 13:52:12,516] - EntityApp::checkTickPeriod: tick took 6075% (6.07 seconds)!
  INFO root [0x00002390] [2018-05-06 13:52:13,542] - Components::findComponents: find logger(1)...
  INFO root [0x00002390] [2018-05-06 13:52:13,556] - Components::findComponents: found logger, addr:192.168.0.106:50447
  INFO root [0x00002390] [2018-05-06 13:52:13,556] - Components::addComponent: reset orderLog, uid=6087!
  INFO root [0x00002390] [2018-05-06 13:52:13,557] - Components::addComponent[logger], uid=6087, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [0x00002390] [2018-05-06 13:52:13,559] - Components::findComponents: find dbmgr(2)...
  WARN root [0x00002390] [2018-05-06 13:52:14,661] - EntityApp::checkTickPeriod: tick took 1219% (1.22 seconds)!
  WARN root [0x00002390] [2018-05-06 13:52:14,663] - Forwarding logs to logger[192.168.0.106:50447/0/0/0]...
 ERROR root [0x00002390] [2018-05-06 13:52:42,182] - Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  INFO root [0x00002390] [2018-05-06 13:52:42,187] - The following logs sent to logger failed:
  WARN root [0x00002390] [2018-05-06 13:52:42,187] - ==> [2018-05-06 13:52:42 180] TCPPacketReceiver::processPendingEvents(192.168.0.106:50447): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x00002390] [2018-05-06 13:52:42,187] - ==> [2018-05-06 13:52:42 182] Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  INFO root [0x00002c04] [2018-05-06 13:53:15,832] - 
  INFO root [0x00002c04] [2018-05-06 13:53:15,832] - -----------------------------------------------------------------------------------------


  INFO root [0x00002c04] [2018-05-06 13:53:15,833] - Resmgr::initialize: KBE_ROOT=E:/kbengine/
  INFO root [0x00002c04] [2018-05-06 13:53:15,833] - Resmgr::initialize: KBE_RES_PATH=E:/kbengine/kbe/res/;E:/kbengine/assets/;E:/kbengine/assets/scripts/;E:/kbengine/assets/res/
  INFO root [0x00002c04] [2018-05-06 13:53:15,834] - Resmgr::initialize: KBE_BIN_PATH=E:/kbengine/kbe/bin/server/
  INFO root [0x00002c04] [2018-05-06 13:53:15,836] - NetworkInterface::initialize(INTERNAL): bound to all interfaces with default route interface on 192.168.0.106 ( 192.168.0.106:50520 )
  INFO root [0x00002c04] [2018-05-06 13:53:15,836] - NetworkInterface::initialize(INTERNAL): address 192.168.0.106:50520, SOMAXCONN=5.
  INFO root [0x00002c04] [2018-05-06 13:53:15,836] - server-configs:
  INFO root [0x00002c04] [2018-05-06 13:53:15,837] - 	gameUpdateHertz : 10
  INFO root [0x00002c04] [2018-05-06 13:53:15,837] - 	defaultViewRadius : 80
  INFO root [0x00002c04] [2018-05-06 13:53:15,837] - 	defaultViewHysteresisArea : 5
  INFO root [0x00002c04] [2018-05-06 13:53:15,837] - 	entryScriptFile : kbemain
  INFO root [0x00002c04] [2018-05-06 13:53:15,837] - 	internalAddr : 192.168.0.106:50520
  INFO root [0x00002c04] [2018-05-06 13:53:15,837] - 	componentID : 8001
  INFO root [0x00002c04] [2018-05-06 13:53:17,353] - ---- cellapp Version: 2.0.0. ScriptVersion: 0.1.0. Protocol: 85A70C5E64F896E0F1A60BF405B4EB07. Config: Debug X86. Built: 15:53:53 Apr 29 2018. AppID: 8001. UID: 6316. PID: 11252 ----
  INFO root [0x00002c04] [2018-05-06 13:53:17,355] - Server : Intel(R) Xeon(R) CPU E3-1230 v3 @ 3.30GHz with 4194303kB RAM
  INFO root [0x00002c04] [2018-05-06 13:53:17,355] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [0x00002c04] [2018-05-06 13:53:17,356] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
 DEBUG root [0x00002c04] [2018-05-06 13:53:17,502] - Script::install_py_dlls(): PyInit_pyexpat
 DEBUG root [0x00002c04] [2018-05-06 13:53:17,503] - Script::install_py_dlls(): PyInit__socket
 DEBUG root [0x00002c04] [2018-05-06 13:53:17,503] - Script::install_py_dlls(): PyInit__ssl
 DEBUG root [0x00002c04] [2018-05-06 13:53:17,507] - Script::install_py_dlls(): PyInit__hashlib
 DEBUG root [0x00002c04] [2018-05-06 13:53:17,508] - Script::install_py_dlls(): PyInit_select
 DEBUG root [0x00002c04] [2018-05-06 13:53:17,508] - Script::install_py_dlls(): PyInit__ctypes
 DEBUG root [0x00002c04] [2018-05-06 13:53:17,508] - Script::install_py_dlls(): PyInit__elementtree
 DEBUG root [0x00002c04] [2018-05-06 13:53:17,537] - Script::install_py_dlls(): PyInit_unicodedata
  INFO root [0x00002c04] [2018-05-06 13:53:17,564] - Script::install(): is successfully, Python=(3.4.2 (default, Apr 29 2018, 15:53:05) [MSC v.1913 32 bit (Intel)])!
  INFO root [0x00002c04] [2018-05-06 13:53:17,565] - Script::install(): KBExtra is successfully!
 DEBUG root [0x00002c04] [2018-05-06 13:53:17,573] - loaded entity-script:Account(1).
  INFO root [0x00002c04] [2018-05-06 13:53:17,574] - TelnetServer server is running on port 50000
  INFO root [0x00002c04] [2018-05-06 13:53:17,951] - ---- cellapp is running ----
 DEBUG root [0x00002c04] [2018-05-06 13:53:17,951] - Components::process(): Request for the process of identity...
  WARN root [0x00002c04] [2018-05-06 13:53:24,036] - EntityApp::checkTickPeriod: tick took 6085% (6.09 seconds)!
  INFO root [0x00002c04] [2018-05-06 13:53:25,051] - Components::findComponents: find logger(1)...
  INFO root [0x00002c04] [2018-05-06 13:53:25,127] - Components::findComponents: found logger, addr:192.168.0.106:50515
  INFO root [0x00002c04] [2018-05-06 13:53:25,127] - Components::addComponent: reset orderLog, uid=6316!
  INFO root [0x00002c04] [2018-05-06 13:53:25,127] - Components::addComponent[logger], uid=6316, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [0x00002c04] [2018-05-06 13:53:25,128] - Components::findComponents: find dbmgr(2)...
  WARN root [0x00002c04] [2018-05-06 13:53:26,229] - EntityApp::checkTickPeriod: tick took 1278% (1.28 seconds)!
  WARN root [0x00002c04] [2018-05-06 13:53:26,230] - Forwarding logs to logger[192.168.0.106:50515/0/0/0]...
 ERROR root [0x00002c04] [2018-05-06 13:54:14,285] - Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  INFO root [0x00002c04] [2018-05-06 13:54:14,296] - The following logs sent to logger failed:
  WARN root [0x00002c04] [2018-05-06 13:54:14,296] - ==> [2018-05-06 13:54:14 280] TCPPacketReceiver::processPendingEvents(192.168.0.106:50515): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x00002c04] [2018-05-06 13:54:14,296] - ==> [2018-05-06 13:54:14 285] Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  INFO root [0x00002f18] [2018-05-06 13:56:01,798] - 
  INFO root [0x00002f18] [2018-05-06 13:56:01,799] - -----------------------------------------------------------------------------------------


  INFO root [0x00002f18] [2018-05-06 13:56:01,799] - Resmgr::initialize: KBE_ROOT=E:/kbengine/
  INFO root [0x00002f18] [2018-05-06 13:56:01,800] - Resmgr::initialize: KBE_RES_PATH=E:/kbengine/kbe/res/;E:/kbengine/assets/;E:/kbengine/assets/scripts/;E:/kbengine/assets/res/
  INFO root [0x00002f18] [2018-05-06 13:56:01,800] - Resmgr::initialize: KBE_BIN_PATH=E:/kbengine/kbe/bin/server/
  INFO root [0x00002f18] [2018-05-06 13:56:01,802] - NetworkInterface::initialize(INTERNAL): bound to all interfaces with default route interface on 192.168.0.106 ( 192.168.0.106:50591 )
  INFO root [0x00002f18] [2018-05-06 13:56:01,802] - NetworkInterface::initialize(INTERNAL): address 192.168.0.106:50591, SOMAXCONN=5.
  INFO root [0x00002f18] [2018-05-06 13:56:01,802] - server-configs:
  INFO root [0x00002f18] [2018-05-06 13:56:01,802] - 	gameUpdateHertz : 10
  INFO root [0x00002f18] [2018-05-06 13:56:01,802] - 	defaultViewRadius : 80
  INFO root [0x00002f18] [2018-05-06 13:56:01,802] - 	defaultViewHysteresisArea : 5
  INFO root [0x00002f18] [2018-05-06 13:56:01,802] - 	entryScriptFile : kbemain
  INFO root [0x00002f18] [2018-05-06 13:56:01,802] - 	internalAddr : 192.168.0.106:50591
  INFO root [0x00002f18] [2018-05-06 13:56:01,802] - 	componentID : 8001
  INFO root [0x00002f18] [2018-05-06 13:56:03,309] - ---- cellapp Version: 2.0.0. ScriptVersion: 0.1.0. Protocol: 85A70C5E64F896E0F1A60BF405B4EB07. Config: Debug X86. Built: 15:53:53 Apr 29 2018. AppID: 8001. UID: 6858. PID: 12256 ----
  INFO root [0x00002f18] [2018-05-06 13:56:03,310] - Server : Intel(R) Xeon(R) CPU E3-1230 v3 @ 3.30GHz with 4194303kB RAM
  INFO root [0x00002f18] [2018-05-06 13:56:03,310] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [0x00002f18] [2018-05-06 13:56:03,310] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
 DEBUG root [0x00002f18] [2018-05-06 13:56:03,472] - Script::install_py_dlls(): PyInit_pyexpat
 DEBUG root [0x00002f18] [2018-05-06 13:56:03,472] - Script::install_py_dlls(): PyInit__socket
 DEBUG root [0x00002f18] [2018-05-06 13:56:03,473] - Script::install_py_dlls(): PyInit__ssl
 DEBUG root [0x00002f18] [2018-05-06 13:56:03,476] - Script::install_py_dlls(): PyInit__hashlib
 DEBUG root [0x00002f18] [2018-05-06 13:56:03,476] - Script::install_py_dlls(): PyInit_select
 DEBUG root [0x00002f18] [2018-05-06 13:56:03,476] - Script::install_py_dlls(): PyInit__ctypes
 DEBUG root [0x00002f18] [2018-05-06 13:56:03,476] - Script::install_py_dlls(): PyInit__elementtree
 DEBUG root [0x00002f18] [2018-05-06 13:56:03,503] - Script::install_py_dlls(): PyInit_unicodedata
  INFO root [0x00002f18] [2018-05-06 13:56:03,530] - Script::install(): is successfully, Python=(3.4.2 (default, Apr 29 2018, 15:53:05) [MSC v.1913 32 bit (Intel)])!
  INFO root [0x00002f18] [2018-05-06 13:56:03,530] - Script::install(): KBExtra is successfully!
 DEBUG root [0x00002f18] [2018-05-06 13:56:03,539] - loaded entity-script:Account(1).
  INFO root [0x00002f18] [2018-05-06 13:56:03,540] - TelnetServer server is running on port 50000
  INFO root [0x00002f18] [2018-05-06 13:56:03,851] - ---- cellapp is running ----
 DEBUG root [0x00002f18] [2018-05-06 13:56:03,851] - Components::process(): Request for the process of identity...
  WARN root [0x00002f18] [2018-05-06 13:56:09,906] - EntityApp::checkTickPeriod: tick took 6054% (6.05 seconds)!
  INFO root [0x00002f18] [2018-05-06 13:56:11,001] - Components::findComponents: find logger(1)...
  INFO root [0x00002f18] [2018-05-06 13:56:11,095] - Components::findComponents: found logger, addr:192.168.0.106:50586
  INFO root [0x00002f18] [2018-05-06 13:56:11,096] - Components::addComponent: reset orderLog, uid=6858!
  INFO root [0x00002f18] [2018-05-06 13:56:11,096] - Components::addComponent[logger], uid=6858, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [0x00002f18] [2018-05-06 13:56:11,097] - Components::findComponents: find dbmgr(2)...
  INFO root [0x00002f18] [2018-05-06 13:56:11,204] - Components::findComponents: found dbmgr, addr:192.168.0.106:50585
  INFO root [0x00002f18] [2018-05-06 13:56:11,205] - Components::addComponent: reset orderLog, uid=6858!
  INFO root [0x00002f18] [2018-05-06 13:56:11,205] - Components::addComponent[dbmgr], uid=6858, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [0x00002f18] [2018-05-06 13:56:11,206] - Components::findComponents: found dbmgr, addr:192.168.0.106:50585
  WARN root [0x00002f18] [2018-05-06 13:56:12,206] - Forwarding logs to logger[192.168.0.106:50586/0/0/0]...
 ERROR root [0x00002f18] [2018-05-06 13:56:42,786] - Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  INFO root [0x00002f18] [2018-05-06 13:56:42,796] - The following logs sent to logger failed:
  WARN root [0x00002f18] [2018-05-06 13:56:42,796] - ==> [2018-05-06 13:56:42 781] TCPPacketReceiver::processPendingEvents(192.168.0.106:50586): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x00002f18] [2018-05-06 13:56:42,796] - ==> [2018-05-06 13:56:42 787] Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  WARN root [0x00002f18] [2018-05-06 13:56:42,838] - TCPPacketReceiver::processPendingEvents(192.168.0.106:50585): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x00002f18] [2018-05-06 13:56:42,841] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit! errorno=10054
  INFO root [0x00002f18] [2018-05-06 13:56:42,875] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [0x00002f18] [2018-05-06 13:56:42,879] - Shutdowner::onShutdown: shutting down
  INFO root [0x00002f18] [2018-05-06 13:56:42,879] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [0x00002f18] [2018-05-06 13:56:42,879] - EntityRef::destroyObjPool(): size 0.
 DEBUG root [0x00002f18] [2018-05-06 13:56:42,879] - Witness::destroyObjPool(): size 0.
 DEBUG root [0x00002f18] [2018-05-06 13:56:42,888] - Script::uninstall_py_dlls(): PyInit_pyexpat
 DEBUG root [0x00002f18] [2018-05-06 13:56:42,888] - Script::uninstall_py_dlls(): PyInit__socket
 DEBUG root [0x00002f18] [2018-05-06 13:56:42,888] - Script::uninstall_py_dlls(): PyInit__ssl
 DEBUG root [0x00002f18] [2018-05-06 13:56:42,888] - Script::uninstall_py_dlls(): PyInit__hashlib
 DEBUG root [0x00002f18] [2018-05-06 13:56:42,888] - Script::uninstall_py_dlls(): PyInit_select
 DEBUG root [0x00002f18] [2018-05-06 13:56:42,888] - Script::uninstall_py_dlls(): PyInit__ctypes
 DEBUG root [0x00002f18] [2018-05-06 13:56:42,888] - Script::uninstall_py_dlls(): PyInit__elementtree
 DEBUG root [0x00002f18] [2018-05-06 13:56:42,888] - Script::uninstall_py_dlls(): PyInit_unicodedata
  INFO root [0x00002f18] [2018-05-06 13:56:42,896] - Script::uninstall(): is successfully!
 DEBUG root [0x00002f18] [2018-05-06 13:56:42,897] - ThreadPool::destroy(): starting size 2.
  INFO root [0x00002abc] [2018-05-06 13:56:44,013] - 
  INFO root [0x00002abc] [2018-05-06 13:56:44,013] - -----------------------------------------------------------------------------------------


  INFO root [0x00002abc] [2018-05-06 13:56:44,014] - Resmgr::initialize: KBE_ROOT=E:/kbengine/
  INFO root [0x00002abc] [2018-05-06 13:56:44,014] - Resmgr::initialize: KBE_RES_PATH=E:/kbengine/kbe/res/;E:/kbengine/assets/;E:/kbengine/assets/scripts/;E:/kbengine/assets/res/
  INFO root [0x00002abc] [2018-05-06 13:56:44,015] - Resmgr::initialize: KBE_BIN_PATH=E:/kbengine/kbe/bin/server/
  INFO root [0x00002abc] [2018-05-06 13:56:44,017] - NetworkInterface::initialize(INTERNAL): bound to all interfaces with default route interface on 192.168.0.106 ( 192.168.0.106:50626 )
  INFO root [0x00002abc] [2018-05-06 13:56:44,017] - NetworkInterface::initialize(INTERNAL): address 192.168.0.106:50626, SOMAXCONN=5.
  INFO root [0x00002abc] [2018-05-06 13:56:44,018] - server-configs:
  INFO root [0x00002abc] [2018-05-06 13:56:44,018] - 	gameUpdateHertz : 10
  INFO root [0x00002abc] [2018-05-06 13:56:44,018] - 	defaultViewRadius : 80
  INFO root [0x00002abc] [2018-05-06 13:56:44,018] - 	defaultViewHysteresisArea : 5
  INFO root [0x00002abc] [2018-05-06 13:56:44,018] - 	entryScriptFile : kbemain
  INFO root [0x00002abc] [2018-05-06 13:56:44,018] - 	internalAddr : 192.168.0.106:50626
  INFO root [0x00002abc] [2018-05-06 13:56:44,018] - 	componentID : 8001
  INFO root [0x00002abc] [2018-05-06 13:56:45,527] - ---- cellapp Version: 2.0.0. ScriptVersion: 0.1.0. Protocol: 85A70C5E64F896E0F1A60BF405B4EB07. Config: Debug X86. Built: 15:53:53 Apr 29 2018. AppID: 8001. UID: 6995. PID: 10808 ----
  INFO root [0x00002abc] [2018-05-06 13:56:45,528] - Server : Intel(R) Xeon(R) CPU E3-1230 v3 @ 3.30GHz with 4194303kB RAM
  INFO root [0x00002abc] [2018-05-06 13:56:45,528] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [0x00002abc] [2018-05-06 13:56:45,529] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
 DEBUG root [0x00002abc] [2018-05-06 13:56:45,673] - Script::install_py_dlls(): PyInit_pyexpat
 DEBUG root [0x00002abc] [2018-05-06 13:56:45,674] - Script::install_py_dlls(): PyInit__socket
 DEBUG root [0x00002abc] [2018-05-06 13:56:45,674] - Script::install_py_dlls(): PyInit__ssl
 DEBUG root [0x00002abc] [2018-05-06 13:56:45,677] - Script::install_py_dlls(): PyInit__hashlib
 DEBUG root [0x00002abc] [2018-05-06 13:56:45,677] - Script::install_py_dlls(): PyInit_select
 DEBUG root [0x00002abc] [2018-05-06 13:56:45,677] - Script::install_py_dlls(): PyInit__ctypes
 DEBUG root [0x00002abc] [2018-05-06 13:56:45,678] - Script::install_py_dlls(): PyInit__elementtree
 DEBUG root [0x00002abc] [2018-05-06 13:56:45,713] - Script::install_py_dlls(): PyInit_unicodedata
  INFO root [0x00002abc] [2018-05-06 13:56:45,758] - Script::install(): is successfully, Python=(3.4.2 (default, Apr 29 2018, 15:53:05) [MSC v.1913 32 bit (Intel)])!
  INFO root [0x00002abc] [2018-05-06 13:56:45,758] - Script::install(): KBExtra is successfully!
 DEBUG root [0x00002abc] [2018-05-06 13:56:45,773] - loaded entity-script:Account(1).
  INFO root [0x00002abc] [2018-05-06 13:56:45,775] - TelnetServer server is running on port 50000
  INFO root [0x00002abc] [2018-05-06 13:56:46,131] - ---- cellapp is running ----
 DEBUG root [0x00002abc] [2018-05-06 13:56:46,131] - Components::process(): Request for the process of identity...
  WARN root [0x00002abc] [2018-05-06 13:56:52,208] - EntityApp::checkTickPeriod: tick took 6077% (6.08 seconds)!
  INFO root [0x00002abc] [2018-05-06 13:56:53,231] - Components::findComponents: find logger(1)...
  INFO root [0x00002abc] [2018-05-06 13:56:53,305] - Components::findComponents: found logger, addr:192.168.0.106:50625
  INFO root [0x00002abc] [2018-05-06 13:56:53,306] - Components::addComponent: reset orderLog, uid=6995!
  INFO root [0x00002abc] [2018-05-06 13:56:53,306] - Components::addComponent[logger], uid=6995, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [0x00002abc] [2018-05-06 13:56:53,307] - Components::findComponents: find dbmgr(2)...
  INFO root [0x00002abc] [2018-05-06 13:56:53,410] - Components::findComponents: found dbmgr, addr:192.168.0.106:50621
  INFO root [0x00002abc] [2018-05-06 13:56:53,411] - Components::addComponent: reset orderLog, uid=6995!
  INFO root [0x00002abc] [2018-05-06 13:56:53,411] - Components::addComponent[dbmgr], uid=6995, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [0x00002abc] [2018-05-06 13:56:53,412] - Components::findComponents: found dbmgr, addr:192.168.0.106:50621
  WARN root [0x00002abc] [2018-05-06 13:56:54,412] - EntityApp::checkTickPeriod: tick took 1280% (1.28 seconds)!
  WARN root [0x00002abc] [2018-05-06 13:56:54,414] - Forwarding logs to logger[192.168.0.106:50625/0/0/0]...
 ERROR root [0x00002abc] [2018-05-06 14:06:40,557] - Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  INFO root [0x00002abc] [2018-05-06 14:06:40,564] - The following logs sent to logger failed:
  WARN root [0x00002abc] [2018-05-06 14:06:40,564] - ==> [2018-05-06 14:06:40 554] TCPPacketReceiver::processPendingEvents(192.168.0.106:50625): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x00002abc] [2018-05-06 14:06:40,564] - ==> [2018-05-06 14:06:40 558] Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  WARN root [0x00002abc] [2018-05-06 14:06:40,609] - TCPPacketReceiver::processPendingEvents(192.168.0.106:50621): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x00002abc] [2018-05-06 14:06:40,611] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit! errorno=10054
  INFO root [0x00002abc] [2018-05-06 14:06:40,616] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [0x00002abc] [2018-05-06 14:06:40,621] - Shutdowner::onShutdown: shutting down
  INFO root [0x00002abc] [2018-05-06 14:06:40,621] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [0x00002abc] [2018-05-06 14:06:40,621] - EntityRef::destroyObjPool(): size 0.
 DEBUG root [0x00002abc] [2018-05-06 14:06:40,621] - Witness::destroyObjPool(): size 0.
 DEBUG root [0x00002abc] [2018-05-06 14:06:40,629] - Script::uninstall_py_dlls(): PyInit_pyexpat
 DEBUG root [0x00002abc] [2018-05-06 14:06:40,630] - Script::uninstall_py_dlls(): PyInit__socket
 DEBUG root [0x00002abc] [2018-05-06 14:06:40,630] - Script::uninstall_py_dlls(): PyInit__ssl
 DEBUG root [0x00002abc] [2018-05-06 14:06:40,630] - Script::uninstall_py_dlls(): PyInit__hashlib
 DEBUG root [0x00002abc] [2018-05-06 14:06:40,630] - Script::uninstall_py_dlls(): PyInit_select
 DEBUG root [0x00002abc] [2018-05-06 14:06:40,630] - Script::uninstall_py_dlls(): PyInit__ctypes
 DEBUG root [0x00002abc] [2018-05-06 14:06:40,630] - Script::uninstall_py_dlls(): PyInit__elementtree
 DEBUG root [0x00002abc] [2018-05-06 14:06:40,630] - Script::uninstall_py_dlls(): PyInit_unicodedata
  INFO root [0x00002abc] [2018-05-06 14:06:40,639] - Script::uninstall(): is successfully!
 DEBUG root [0x00002abc] [2018-05-06 14:06:40,639] - ThreadPool::destroy(): starting size 2.
  INFO root [0x00000d2c] [2018-05-06 14:06:41,740] - 
  INFO root [0x00000d2c] [2018-05-06 14:06:41,741] - -----------------------------------------------------------------------------------------


  INFO root [0x00000d2c] [2018-05-06 14:06:41,741] - Resmgr::initialize: KBE_ROOT=E:/kbengine/
  INFO root [0x00000d2c] [2018-05-06 14:06:41,741] - Resmgr::initialize: KBE_RES_PATH=E:/kbengine/kbe/res/;E:/kbengine/assets/;E:/kbengine/assets/scripts/;E:/kbengine/assets/res/
  INFO root [0x00000d2c] [2018-05-06 14:06:41,741] - Resmgr::initialize: KBE_BIN_PATH=E:/kbengine/kbe/bin/server/
  INFO root [0x00000d2c] [2018-05-06 14:06:41,748] - NetworkInterface::initialize(INTERNAL): bound to all interfaces with default route interface on 192.168.0.106 ( 192.168.0.106:50826 )
  INFO root [0x00000d2c] [2018-05-06 14:06:41,748] - NetworkInterface::initialize(INTERNAL): address 192.168.0.106:50826, SOMAXCONN=5.
  INFO root [0x00000d2c] [2018-05-06 14:06:41,748] - server-configs:
  INFO root [0x00000d2c] [2018-05-06 14:06:41,748] - 	gameUpdateHertz : 10
  INFO root [0x00000d2c] [2018-05-06 14:06:41,748] - 	defaultViewRadius : 80
  INFO root [0x00000d2c] [2018-05-06 14:06:41,749] - 	defaultViewHysteresisArea : 5
  INFO root [0x00000d2c] [2018-05-06 14:06:41,749] - 	entryScriptFile : kbemain
  INFO root [0x00000d2c] [2018-05-06 14:06:41,749] - 	internalAddr : 192.168.0.106:50826
  INFO root [0x00000d2c] [2018-05-06 14:06:41,749] - 	componentID : 8001
  WARN root [0x00000d2c] [2018-05-06 14:06:42,060] - BundleBroadcast::BundleBroadcast: Cannot bind to port 20570, errorno=10048
  INFO root [0x00000d2c] [2018-05-06 14:06:42,064] - ---- cellapp Version: 2.0.0. ScriptVersion: 0.1.0. Protocol: 85A70C5E64F896E0F1A60BF405B4EB07. Config: Debug X86. Built: 15:53:53 Apr 29 2018. AppID: 8001. UID: 8948. PID: 11392 ----
  INFO root [0x00000d2c] [2018-05-06 14:06:42,064] - Server : Intel(R) Xeon(R) CPU E3-1230 v3 @ 3.30GHz with 4194303kB RAM
  INFO root [0x00000d2c] [2018-05-06 14:06:42,064] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [0x00000d2c] [2018-05-06 14:06:42,064] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
 DEBUG root [0x00000d2c] [2018-05-06 14:06:42,196] - Script::install_py_dlls(): PyInit_pyexpat
 DEBUG root [0x00000d2c] [2018-05-06 14:06:42,197] - Script::install_py_dlls(): PyInit__socket
 DEBUG root [0x00000d2c] [2018-05-06 14:06:42,197] - Script::install_py_dlls(): PyInit__ssl
 DEBUG root [0x00000d2c] [2018-05-06 14:06:42,201] - Script::install_py_dlls(): PyInit__hashlib
 DEBUG root [0x00000d2c] [2018-05-06 14:06:42,202] - Script::install_py_dlls(): PyInit_select
 DEBUG root [0x00000d2c] [2018-05-06 14:06:42,202] - Script::install_py_dlls(): PyInit__ctypes
 DEBUG root [0x00000d2c] [2018-05-06 14:06:42,203] - Script::install_py_dlls(): PyInit__elementtree
 DEBUG root [0x00000d2c] [2018-05-06 14:06:42,233] - Script::install_py_dlls(): PyInit_unicodedata
  INFO root [0x00000d2c] [2018-05-06 14:06:42,266] - Script::install(): is successfully, Python=(3.4.2 (default, Apr 29 2018, 15:53:05) [MSC v.1913 32 bit (Intel)])!
  INFO root [0x00000d2c] [2018-05-06 14:06:42,266] - Script::install(): KBExtra is successfully!
 DEBUG root [0x00000d2c] [2018-05-06 14:06:42,276] - loaded entity-script:Account(1).
  INFO root [0x00000d2c] [2018-05-06 14:06:42,277] - TelnetServer server is running on port 50000
  INFO root [0x00000d2c] [2018-05-06 14:06:42,604] - ---- cellapp is running ----
 DEBUG root [0x00000d2c] [2018-05-06 14:06:42,604] - Components::process(): Request for the process of identity...
  WARN root [0x00000d2c] [2018-05-06 14:06:48,684] - EntityApp::checkTickPeriod: tick took 6080% (6.08 seconds)!
 DEBUG root [0x00000d2c] [2018-05-06 14:06:49,413] - Cellapp::lookApp: 192.168.0.106:50840/0/0/0
  INFO root [0x00000d2c] [2018-05-06 14:06:49,704] - Components::findComponents: find logger(1)...
  INFO root [0x00000d2c] [2018-05-06 14:06:49,793] - Components::findComponents: found logger, addr:192.168.0.106:50825
  INFO root [0x00000d2c] [2018-05-06 14:06:49,794] - Components::addComponent: reset orderLog, uid=8948!
  INFO root [0x00000d2c] [2018-05-06 14:06:49,794] - Components::addComponent[logger], uid=8948, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [0x00000d2c] [2018-05-06 14:06:49,796] - Components::findComponents: find dbmgr(2)...
  INFO root [0x00000d2c] [2018-05-06 14:06:49,897] - Components::findComponents: found dbmgr, addr:192.168.0.106:50822
  INFO root [0x00000d2c] [2018-05-06 14:06:49,897] - Components::addComponent: reset orderLog, uid=8948!
  INFO root [0x00000d2c] [2018-05-06 14:06:49,898] - Components::addComponent[dbmgr], uid=8948, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [0x00000d2c] [2018-05-06 14:06:49,898] - Components::findComponents: found dbmgr, addr:192.168.0.106:50822
  WARN root [0x00000d2c] [2018-05-06 14:06:50,898] - EntityApp::checkTickPeriod: tick took 1294% (1.29 seconds)!
  WARN root [0x00000d2c] [2018-05-06 14:06:50,899] - Forwarding logs to logger[192.168.0.106:50825/0/0/0]...
